node/resolvers/providerLogo.ts

```diff
@@ -1,16 +1,30 @@
+/*
+The `src` have two variations, based on weather or not the app is in development (linked).
+-  Development/linked: https://w0geo--checkoutio.myvtex.com/_v/private/assets/v1/linked/vtex.google-geolocation-resolver@0.1.0/public/logo.png
+- Production/released: https://w0geo--checkoutio.myvtex.com/_v/public/assets/v1/published/vtex.google-geolocation-resolver@0.1.0/public/logo.png
+*/
 import { Image } from 'vtex.geolocation-graphql-interface'
 
-// must be the same as assets/logo.png
-const SRC_URL =
-  'https://user-images.githubusercontent.com/26108090/100493516-45802680-3116-11eb-9028-06c336a587a7.png'
-
 const ALT_TEXT = 'Google logo'
```

Comment:
we should add a todo comment here to translate this string later (don't know how yet)

Reply:
Well noticed 🤔, will add